index.js

```diff
@@ -1,7 +1,30 @@
 // Iteration 1: Names and Input
 
+var hacker1 = { firstName: 'Jeremy', lastName: 'Couderc' };
+var message = 'The drivers name is ' + hacker1.firstName + ' ' + hacker1.lastName + '!!';
+console.log(message);
 
 // Iteration 2: Conditionals
 
+const message = `The driver has the longest name, it has XX characters`;
+
+console.log(message.length);
+
+const  XX = '53';
+const message2 = `The driver has the longest name, it has ${XX} characters`;
```

Comment:
Here we possibly could have used `const message2 = `The driver has the longest name, it has ${hacker1.firstName.length} characters`;` to get the length of that string